### PR TITLE
Task/basic structure app

### PR DESCRIPTION
**Pull request by AbdalazizAbdallah**
Task/basic structure app Branch

works :

add classes constants app like image class, sizes, styles, colors, strings, and themes.
add localizations for the app Arabic and English.
add storages folders remote, local
add route class contains the name of pages and list of GetPages

install packages : 
  get: ^4.6.5
  flutter_screenutil: ^5.5.4
  dio: ^4.0.6
  pretty_dio_logger: ^1.2.0-beta-1

### CODE REVIEW

File: lib/core/constants/colors_app.dart [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import 'package:flutter/material.dart';
+
+class ColorsApp {
+
+  static late Color primary;
```
Comment (fahjan) [inline lib/core/constants/colors_app.dart:5]: prefer to write:
// TODO(team member): assign colors

File: lib/core/localizations/ar.dart [hunk 2]
```diff
@@ -0,0 +1,9 @@
+
+
```
Comment (fahjan) [inline lib/core/localizations/ar.dart:2]: class with one variable not needed, you can use variable directly

File: lib/core/localizations/en.dart [hunk 3]
```diff
@@ -0,0 +1,5 @@
+class En {
```
Comment (fahjan) [inline lib/core/localizations/en.dart:1]: class with one variable not needed, you can use variable directly

File: lib/core/localizations/ar.dart [hunk 4]
```diff
@@ -0,0 +1,9 @@
...
```
Comment (fahjan) [inline lib/core/localizations/ar.dart:1]: there is no need to empty lines

File: lib/main.dart [hunk 5]
```diff
@@ -10,106 +17,23 @@ class MyApp extends StatelessWidget {
...
+    return ScreenUtilInit(
+      designSize: const Size(360, 690),
+      minTextAdapt: true,
+      splitScreenMode: true,
+      builder: (_, child) {
+        return GetMaterialApp(
+          title: StringsApp.titleApp,
+          theme: ThemesApp.light,
+          translations: LocalizationApp(),
+          locale: const Locale('en','US'),
```
Comment (fahjan) [inline lib/main.dart:29]: use localized variable please

File: pubspec.lock [hunk 6]
```diff
@@ -7,7 +7,7 @@ packages:
       name: async
```
Comment (fahjan) [inline pubspec.lock:7]: please, ignore this file from git

File: pubspec.yaml [hunk 7]
```diff
@@ -34,6 +34,10 @@ dependencies:
   # The following adds the Cupertino Icons font to your application.
   # Use with the CupertinoIcons class for iOS style icons.
   cupertino_icons: ^1.0.2
+  get: ^4.6.5
+  flutter_screenutil: ^5.5.4
+  dio: ^4.0.6
+  pretty_dio_logger: ^1.2.0-beta-1
```
Comment (fahjan) [inline pubspec.yaml:41]: need onesignal library

Comment (fahjan) [inline pubspec.yaml:41]: need onesignal library

File: lib/main.dart (hunk 5 above)
Comment (AbdalazizAbdallah) [reply lib/main.dart:29]: once start in the project we will create controller for this also for theme

**Review by fahjan — APPROVED** [review]

**Pull request merged**